pyproject.toml

```diff
@@ -41,6 +41,7 @@ dev = [
     "hashin ==0.17.0",
     "mypy >=0.921,<=0.961",
     "pylint >=2.9.3,<=2.14.4",
+    "commitizen ==2.28.0",
```

Comment:
Why not `"commitizen >=2.28.0,<3"`?

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/263/commits/40890a364077b409fa3176447731772daccde871 pins commitizen to v2.28.0 inside the Action itself. For consistency and because we run commitizen as a standalone app, I pinned it to a version instead of a range.

Reply:
As discussed, please turn that to a comment in code.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/263/commits/0a1d3cdf48b3e7b18360d6d3264a4ef6759d54c8.